test/index-test.js

```diff
@@ -12,8 +23,9 @@ const collect = (stream, scheduler) => {
     .then(() => eventValues)
 }
 
+const scheduler = newDefaultScheduler()
```

Comment:
As a default, we tend to prefer creating a scheduler per test (they're not expensive to create) to ensure test isolation, unless there's a compelling reason to do something different.  Can this be created inside the `test` function (and also inside the verifyHold function as before?

Reply:
Sure, fixed

Reply:
👍